### DISCUSSION TITLE
How do i implement vanilla enchants

### DISCUSSION THREAD

**Discussion by SnowmanFTW** [Minestom/Minestom#1166] [Help] [answered]
~~Like for example efficiency, if i want it to be like vanilla, how would i go about adding it to an items and making the blocks break faster.~~

Nvm, got it, but how do you hide the enchants?

**Comment by breitwan** [comment, marked as answer]
What do you mean by "hide the enchants"? Hide enchants on item in inventory?

```
var stack = ItemStack.of(Material.STICK)
                .withMeta(builder -> builder.enchantment(Enchantment.KNOCKBACK,  (short)2).hideFlag(ItemHideFlag.HIDE_ENCHANTS));
```

**Reply by SnowmanFTW** [reply]
I managed to find a workaround lol. I guess yours will work too but this is what i eventually did:
```
ItemStack pickaxe = ItemStack.builder(Material.DIAMOND_PICKAXE)
.set(Tag.Byte("HideFlags"), (byte) 1)
.meta(builder -> builder.enchantment(Enchantment.EFFICIENCY, (short) 10))
.build();
```
